selfdrive/controls/controlsd.py

```diff
@@ -212,6 +209,14 @@ def set_initial_state(self):
       if any(ps.controlsAllowed for ps in self.sm['pandaStates']):
         self.state = State.enabled
 
+    if SIMULATION:
+      ignore = ['driverCameraState', 'managerState']
+      available_streams = VisionIpcClient.available_streams("camerad", block=False)
+      if len(available_streams) == 1 and VisionStreamType.VISION_STREAM_WIDE_ROAD in available_streams:
+        ignore += ['roadCameraState']
+      self.sm.ignore_alive += ignore
```

Comment:
this block shouldn't go in set_initial_state, it can just go directly in the body where set_initial_state is called